resources/help.txt

```diff
@@ -0,0 +1,5 @@
+****Sanic Bot****
+Hello! I am SanicBot, a bot specially created for the Sanic Framework Discord server!
+***Argument Help:*** `[Mandatory]` `(Optional)`
+>>> ~~---------------~~:tools: Commands :tools:~~----------------~~
+**+git** / **+g**: `[number]` `(repo)` - Retreives an issue or pull request from any repository in the Sanic organization github via number.
```

Comment:
Did we settle on a prefix?

Reply:
Never did, until I saw it settled in the discord.